.eslintrc.json

```diff
@@ -1,21 +1,21 @@
 {
 	"env": {
-		"browser": true
+		"browser": true,
+		"es2022": "true"
```

Comment:
`true`, but I guess it still works with a string

Reply:
https://github.com/SAP/openui5-sample-app/pull/208